### PR TITLE
Bug 1207309 - Disable '<' add test button if adjacent container empty

### PR DESCRIPTION
**Pull request by MikeLing**

### CODE REVIEW

**Comment by MikeLing** [comment]
screen shot 2015-09-23 at 8 01 34 am
 Hope it can fit your mind :)

**Comment by tojonmz** [comment]
@MikeLing Looks fine in local testing. I suppose if we wanted to enhance it further we could:
* Provide a "No added tests to remove" title/tooltip when the container is empty (I defer to @wlach)

And if we wanted to increase the scope of the fix we could:
* Render the lower [>] icon in blue `btn-primary disabled` when no tests are selected in the left list with a "Please select a test", to try to draw attention that is the most important button in the UI other than the exsiting Add+

But I defer to Will here :)

File: ui/partials/perf/testdatachooser.html [hunk 1]
```diff
@@ -28,10 +28,10 @@ <h3 class="modal-title">Add test data</h3>
     </div>
   </div>
   <div class="btn-group-vertical ">
-    <button id="unselect-test" ng-click="unselectTest()" type="button" class="btn btn-xs btn-default">
+    <button id="unselect-test" ng-click="unselectTest()" type="button" class="btn btn-xs btn-default" ng-disabled="!testsToAdd.length">
       <span class="glyphicon glyphicon-chevron-left"></span>
     </button>
-    <button id="select-test" ng-click="selectTest()" type="button" class="btn btn-xs btn-default">
+    <button id="select-test" ng-click="selectTest()" type="button" class="btn btn-xs btn-default" ng-disabled="!unselectedTestList.length">
```
Comment (tojonmz) [inline ui/partials/perf/testdatachooser.html]: These seem to look fine to me. I defer to you guys if you want to wrap the attributes/directives for shorter lines.

**Comment by wlach** [comment]
Merged in c4a067ef99500c84e741a2bf02a0514663fb6b29
